src/components/Navigation.tsx

```diff
@@ -143,18 +155,25 @@ const Navigation: FC = () => {
   return (
     <>
       <header className="l-navigation-bar">
-        <div className="p-panel is-dark">
+        <div
+          className={classnames("p-panel", {
+            "is-light": !isAuthenticated,
+            "is-dark": isAuthenticated,
+          })}
+        >
           <div className="p-panel__header">
             <Logo />
-            <div className="p-panel__controls">
-              <Button
-                dense
-                className="p-panel__toggle"
-                onClick={hardToggleMenu}
-              >
-                Menu
-              </Button>
-            </div>
+            {isAuthenticated && (
+              <div className="p-panel__controls">
+                <Button
+                  dense
+                  className="p-panel__toggle"
+                  onClick={hardToggleMenu}
+                >
+                  Menu
+                </Button>
+              </div>
+            )}
```

Comment:
We shouldn't hide this, it is the only way to open the navigation to go to step 1 / step 2 / back to oidc login on a very small screen.